### PR TITLE
[fix](multi-catalog) when checkProperties failed,will have dirty data

### PR DESCRIPTION
**Pull request by zddr**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.15 seconds
 stream load tsv:          473 seconds loaded 74807831229 Bytes, about 150 MB/s
 stream load json:         24 seconds loaded 2358488459 Bytes, about 93 MB/s
 stream load orc:          75 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230316141559_clickbench_pr_115468.html

**Comment by zddr** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java [hunk 1]
```diff
@@ -465,6 +462,11 @@ public CatalogIf replayCreateCatalog(CatalogLog log) throws DdlException {
         writeLock();
         try {
             CatalogIf catalog = CatalogFactory.constructorFromLog(log);
+            if (Env.getCurrentEnv().isMaster()) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java:465]: Call `checkProperties()` only when this is not a replay thread

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by zddr** [comment]
run buildall

**Pull request merged**